README.md

```diff
@@ -8,11 +10,67 @@ Features planned:
 - Detect XSS
 - Detect directory traversal
 
+Using it like a user:
+`python -m pyt -f example/vulnerable_code/XSS_call.py save -du`
+
+Running the tests: `python -m tests`
+
+Running an individual test file: `python -m unittest tests.import_test`
+
+Running an individual test: `python -m unittest tests.import_test.ImportTest.test_import`
+
 Work in progress
 
 # Contributions
 Join our slack group: https://pyt-dev.slack.com/
 
 [Guidelines](https://github.com/python-security/pyt/blob/master/CONTRIBUTIONS.md)
 
-[![Build Status](https://travis-ci.org/python-security/pyt.svg?branch=master)](https://travis-ci.org/python-security/pyt)
+## Virtual env setup guide
+
+Create a directory to hold the virtual env and project
+
+`mkdir ~/pyt`
+
+`cd ~/pyt`
+
+Clone the project into the directory
+
+`git clone https://github.com/python-security/pyt.git`
```

Comment:
Naw it works, just wanted to make sure you were accounting for it. You could change the README so that it's clear that the directory for the venv doesn't matter